### ISSUE TITLE
Two simple questions about training

### ISSUE THREAD

**Issue by bo-miao** [hkchengrex/STCN#90]
Hi Ho Kei,

May I have two more questions?
I see from the paper that "Batchnorm layers are set to eval mode during training". However, I did not find any code to freeze these layers for both ResNet50 and ResNet18. Is that an unnecessary part? 
In addition, have you tried to freeze all parameters of ResNet to train the model (Just curious if it can get similar performance, as this could help the model converge faster)?

Thank you.
